components/layout/Inventory.tsx

```diff
@@ -1,61 +1,46 @@
 
-  const connect = async (): Promise<void> => {
-    setConnecting(true);
-    setAccount(await fetchAccount());
-  };
+  useEffect(() => {
+    if (walletAddress) {
+      refreshBalances();
+    }
+  }, [walletAddress]);
 
-  if (!PROD) {
-    useEffect(() => {
-      connect();
-    }, []);
+  if (!walletAddress || !balances || !managedContracts || !namedContracts) {
+    return null;
   }
 
-  if (!balances || !contracts || !account)
-    return (
-      <div className="pointer-events-auto">
-        <button
-          className="m-0 rounded-none rounded-t-lg bg-accent text-paper opacity-100 hover:bg-accent hover:drop-shadow-accent"
-          disabled={connecting}
-          onClick={() => connect()}
-        >
-          Connect Wallet
-        </button>
-      </div>
-    );
-
-  const address = account.address;
-
   return (
-    <div className="pointer-events-auto max-w-screen-lg translate-y-[71%] rounded-t-lg border border-b-0 border-solid border-accent/60 bg-paper transition-transform duration-500 ease-out hover:translate-y-0">
+    <div className="pointer-events-auto max-w-screen-lg translate-y-[65%] rounded-t-lg border border-b-0 border-solid border-accent/60 bg-paper transition-transform duration-500 ease-out hover:translate-y-0">
       <div className="relative mb-2 flex text-left uppercase tracking-widest">
         <div className="mt-2 ml-2 flex flex-grow items-center pl-2 text-xs">My inventory</div>
         <Network />
       </div>
-      <a className="mb-2 block font-mono text-xs text-white/50" href={`https://etherscan.io/address/${address}`}>
-        {address}
-      </a>
       <div className="flex justify-center px-2 pb-2">
         <div className="grid grid-cols-4 gap-2">
-          <Token token="uAD" balance={balances.uad} accountAddr={address} tokenAddr={contracts.uad.address} />
-          <Token token="uAR" balance={balances.uar} accountAddr={address} tokenAddr={contracts.uar.address} />
-          <Token token="uDEBT" balance={balances.debtCoupon} accountAddr={address} tokenAddr={contracts.debtCouponToken.address} />
-          <Token token="UBQ" balance={balances.ubq} accountAddr={address} tokenAddr={contracts.ugov.address} />
-          <Token token="3CRV" balance={balances.crv} accountAddr={address} tokenAddr={contracts.crvToken.address} />
-          <Token token="uAD3CRV-f" balance={balances.uad3crv} accountAddr={address} tokenAddr={contracts.metaPool.address} />
-          <Token token="USDC" balance={balances.usdc} accountAddr={address} tokenAddr={contracts.usdc.address} decimals={6} />
+          <Token token="uAD" balance={balances.uad} accountAddr={walletAddress} tokenAddr={managedContracts.uad.address} />
+          <Token token="uCR" balance={balances.ucr} accountAddr={walletAddress} tokenAddr={managedContracts.uar.address} />
+          <Token token="uCR-NFT" balance={balances.ucrNft} accountAddr={walletAddress} tokenAddr={managedContracts.debtCouponToken.address} />
```

Comment:
@Zequez can you make sure that the icons are correct 
<img width="222" alt="image" src="https://user-images.githubusercontent.com/4975670/169941806-22f264ea-50d0-40d6-8370-cf120cdf3710.png">
#146